src/pykeen/models/unimodal/conv_e.py

```diff
@@ -291,15 +242,15 @@ def _convolve_entity_relation(self, h: torch.LongTensor, r: torch.LongTensor) ->
 
         try:
             # batch_size, num_input_channels, 2*height, width
-            if self.apply_batch_normalization:
```

Comment:
can we delete this flag?

Reply:
🤷 this change just makes it correct with typing. I wasn't going in to change how stuff works